docs/_docs/installation/macos.md

```diff
@@ -3,155 +3,28 @@ title: Jekyll on macOS
-To run the latest Ruby version you need to install it through [Homebrew](https://brew.sh).
-
-```sh
-# Install Homebrew
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-
-# Install Ruby
-brew install ruby
-```
-
-Add the brew ruby and gems path to your shell configuration:
-
-```bash
-# If you're using Zsh
-echo 'export PATH="/usr/local/opt/ruby/bin:/usr/local/lib/ruby/gems/3.0.0/bin:$PATH"' >> ~/.zshrc
-
-# If you're using Bash
-echo 'export PATH="/usr/local/opt/ruby/bin:/usr/local/lib/ruby/gems/3.0.0/bin:$PATH"' >> ~/.bash_profile
-
-# Unsure which shell you are using? Type
-echo $SHELL
-```
-
-Relaunch your terminal and check your Ruby setup:
-
-```sh
-which ruby
-# /usr/local/opt/ruby/bin/ruby
-
-ruby -v
-{{ site.data.ruby.current_version_output }}
-```
-
-You're now running the current stable version of Ruby!
-
-### With rbenv {#rbenv}
-
-People often use [rbenv](https://github.com/rbenv/rbenv) to manage multiple
-Ruby versions. This is very useful when you need to be able to run a given Ruby version on a project.
-
-```sh
-# Install Homebrew
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-
-# Install rbenv and ruby-build
-brew install rbenv
-
-# Set up rbenv integration with your shell
-rbenv init
-
-# Check your installation
-curl -fsSL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-doctor | bash
-```
-
-Restart your terminal to apply your changes.
-Next, you can install the Ruby version you want. Let's install the latest stable version:
+To install Jekyll on macOS, you need a proper Ruby development environment. While macOS comes preinstalled with Ruby, we don't recommend using that version to install Jekyll. This external article goes over the various reasons [why you shouldn't use the system Ruby](https://www.moncefbelyamani.com/why-you-shouldn-t-use-the-system-ruby-to-install-gems-on-a-mac/).
 
-```sh
-rbenv install {{ site.data.ruby.current_version }}
-rbenv global {{ site.data.ruby.current_version }}
-ruby -v
-{{ site.data.ruby.current_version_output }}
-```
+Instead, you'll need to install a separate and newer version of Ruby using a version manager such as [asdf](https://asdf-vm.com/), [chruby](https://github.com/postmodern/chruby), [rbenv](https://github.com/rbenv/rbenv), or [rvm](https://rvm.io/). Version managers allow you to easily install multiple versions of Ruby, and switch between them.
 
-That's it! Head over to [rbenv command references](https://github.com/rbenv/rbenv#command-reference) to learn how to use different versions of Ruby in your projects.
+We recommend `chruby` because it's the most reliable and easiest to use. Follow this external guide for [installing Jekyll on a Mac](https://www.moncefbelyamani.com/how-to-install-jekyll-on-a-mac-the-easy-way/) and setting up a proper Ruby environment with `chruby`.
 
 ## Install Jekyll
 
-After installing Ruby, install Jekyll and Bundler.
-
-### Local Install
```

Comment:
This entire local install section only applies when using the system Ruby. When using a version manager like `chruby`, it takes care of setting the `PATH`, and therefore we can eliminate this whole section, and the complexity of having to set the `PATH` and remembering to update it to match the Ruby version.